corehq/apps/linked_domain/tests/test_linked_apps.py

```diff
@@ -303,7 +303,7 @@ def test_fetch_missing_media(self):
         remote_details = RemoteLinkDetails(
             'http://localhost:8000', 'user', 'key'
         )
-        data = 'this is a test'
+        data = b'this is a test'
```

Comment:
Would it make sense to put some bytes that can't be encoded as utf-8 in this string to make it clear it's not text data?

Reply:
How about I comment this line? (I'm not sure it'd be obvious to me as a reader what bytes can't be encoded, or why this string should contain them.)